reasoner/resolution/resolver/ConjunctionResolver.java

```diff
@@ -313,30 +287,27 @@ public Nested(Actor<Nested> self, Conjunction conjunction, Actor<ResolutionRecor
         @Override
         protected void nextAnswer(Request fromUpstream, ResponseProducer responseProducer, int iteration) {
             if (responseProducer.hasUpstreamAnswer()) {
-                AnswerState.UpstreamVars.Derived upstreamAnswer = responseProducer.upstreamAnswers().next();
-                responseProducer.recordProduced(upstreamAnswer.withInitialFiltered());
-                ResolutionAnswer answer = new ResolutionAnswer(upstreamAnswer, conjunction.toString(),
-                                                               ResolutionAnswer.Derivation.EMPTY, self(), false);
-                respondToUpstream(Response.Answer.create(fromUpstream, answer), iteration);
+                Partial<?> upstreamAnswer = responseProducer.upstreamAnswers().next();
+                responseProducer.recordProduced(upstreamAnswer.conceptMap());
+                answerToUpstream(upstreamAnswer, fromUpstream, iteration);
             } else {
                 if (responseProducer.hasDownstreamProducer()) {
                     requestFromDownstream(responseProducer.nextDownstreamProducer(), fromUpstream, iteration);
                 } else {
-                    respondToUpstream(new Response.Fail(fromUpstream), iteration);
+                    failToUpstream(fromUpstream, iteration);
                 }
             }
         }
 
         @Override
-        protected ResourceIterator<AnswerState.UpstreamVars.Derived> toUpstreamAnswers(Request fromUpstream,
-                                                                                       ResourceIterator<ConceptMap> downstreamConceptMaps) {
-            return downstreamConceptMaps.map(conceptMap -> fromUpstream.partialAnswer().asIdentity()
-                    .aggregateToUpstream(conceptMap, null));
+        protected ResourceIterator<Partial<?>> toUpstreamAnswers(Request fromUpstream,
+                                                                 ResourceIterator<ConceptMap> downstreamConceptMaps) {
+            return downstreamConceptMaps.map(conceptMap -> fromUpstream.partialAnswer().asFiltered().toUpstream(self()));
```

Comment:
since conjunctions don't do traversals anymore, I dont think we need this nor its abstract parent at all :D